upgrading/upgrade-operator.adoc

```diff
@@ -11,14 +11,46 @@ Upgrades through the {rh-rhacs-first} Operator are performed automatically or ma
 
-To roll back an Operator upgrade, you must perform the steps described in one of the following sections. You can roll back an Operator upgrade by using the CLI or the {ocp} web console.
+[IMPORTANT]
+====
+* Because of the database related changes introduced in {product-title-short} 4.0, even if you have selected *Automatic* in the *Update approval* field, you must manully upgrade to {product-title-short} 4.0.
+* You must be using {product-title-short} 3.74 to upgrade to {product-title-short} 4.0. If you are using a version older than 3.74, you must first upgrade to {product-title-short} 3.74 and then upgrade to {product-title-short} 4.0.
+====
 
-include::modules/rollback-operator-upgrades-cli.adoc[leveloffset=+1]
-include::modules/rollback-operator-upgrades-console.adoc[leveloffset=+1]
+include::modules/prepare-operator-upgrades.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
-== Additional resources
+.Additional resources
+* link:https://docs.openshift.com/container-platform/latest/operators/admin/olm-upgrading-operators.html[Updating installed Operators]
+* xref:../backup_and_restore/backing-up-acs.adoc[Backing up {product-title}]
+
+include::modules/operator-upgrade-modify-central-custom-resource.adoc[leveloffset=+1]
+
+include::modules/operator-upgrade-change-subscription-channel.adoc[leveloffset=+1]
+
+[id="rollback-operator-upgrade"]
+== Rolling back an Operator upgrade
+
+To roll back an Operator upgrade, you must perform the steps described in one of the following sections. You can roll back an Operator upgrade by using the CLI or the {ocp} web console.
+
+[NOTE]
+====
+If you are rolling back from {product-title-short} 4.0, you can only rollback to the latest patch release version of {product-title-short} 3.74.
+====
 
+include::modules/rollback-operator-upgrades-cli.adoc[leveloffset=+2]
+include::modules/rollback-operator-upgrades-console.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
 * xref:../installing/installing_ocp/install-central-ocp.adoc#install-central-operator_install-central-ocp[Installing Central using the Operator method]
 * link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/operators/understanding-operators#olm-workflow[Operator Lifecycle Manager workflow]
-* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/operators/administrator-tasks#olm-approving-pending-upgrade_olm-upgrading-operators[Manually approving a pending Operator update]
\ No newline at end of file
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/operators/administrator-tasks#olm-approving-pending-upgrade_olm-upgrading-operators[Manually approving a pending Operator update]
+
+[id="operator-upgrade-troubleshooting_{context}"]
+== Troubleshooting Operator upgrade issues
```

Comment:
@gaurav-nelson I added some more here https://docs.engineering.redhat.com/display/StackRox/Upgrade+to+4.0#Upgradeto4.0-Troubleshooting . Would be very nice if this can be included in the docs.